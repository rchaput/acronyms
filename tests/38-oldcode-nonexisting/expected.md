# List Of Acronyms {#acronyms_HEADER_LOA .loa}

# Introduction {#intro}

The non-existing acronym will be replaced by "??": ??.

Here, with its key: RL.
