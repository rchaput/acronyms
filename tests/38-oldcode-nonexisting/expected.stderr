(W) [acronyms] Acronym key RL not recognized
(W) [acronyms] Acronym key RL not recognized
